common/flags/Flag.java

```diff
@@ -44,6 +48,26 @@
     return new Flag.DoubleFlag(defaultValue);
   }
 
+  public static Flag<List<String>> createStringsListFlag(List<String> defaultValue) {
```

Comment:
I think you can name all of them `create()` since they differ in their inputs.

Reply:
It's impossible to use one method's name for all methods which work with Lists<> because generic types are erased at runtime, so all of these methods compile to the same signature. If we want to use one name we need to think it over how to do it.

Reply:
Interesting, though are you sure about this? See these:
https://stackoverflow.com/questions/3452859/java-generics-code-compiles-with-javac-fails-with-eclipse-helios

https://stackoverflow.com/questions/5527235/type-erasure-and-overloading-in-java-why-does-this-work?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa